### ISSUE TITLE
allowRemove option with multiply FilePonds

### ISSUE THREAD

**Issue by xpert13** [pqina/filepond#712]
**Describe the bug**

When we create multiply FilePond instances and the first of them has disabled "allowRemove" then all of them won't have a Remove button.

**To Reproduce**

```
import "filepond/dist/filepond.min.css";

import { create } from "filepond";

const fileInfo = [
  {
    source: "1",
    options: {
      type: "local",
      file: {
        name: "file.pdf",
        size: 36228,
        type: "application/pdf"
      }
    }
  }
];

create(document.querySelector("#file1"), {
  allowRemove: false,
  files: fileInfo
});

create(document.querySelector("#file2"), {
  allowRemove: true,
  files: fileInfo
});
```
https://codesandbox.io/s/vanilla-filepond-forked-8d0tx?file=/src/index.js:0-443

**Expected behavior**

Each FilePond shows a Remove button depending on its own options.

**Information about your project:**

- Your browser and the version: Chrome 90.0
- Your operating system: macOS 11.0.1
- FilePond version 4.27.1

**Comment by rikschennink** [comment]
Thanks will merge and fix this week.

**Comment by rikschennink** [comment]
Fixed in `4.27.3`
